Source/WebKit/WebProcess/Inspector/ServiceWorkerDebuggableProxy.cpp

```diff
@@ -74,6 +75,17 @@ void ServiceWorkerDebuggableProxy::dispatchMessageFromRemote(String&& message)
         webProcessProxy->send(Messages::WebSWContextManagerConnection::DispatchMessageFromInspector(m_identifier, WTFMove(message)), 0);
 }
 
+#if ENABLE(REMOTE_INSPECTOR_SERVICE_WORKER_AUTO_INSPECTION)
+void ServiceWorkerDebuggableProxy::pauseWaitingForAutomaticInspection()
+{
+    m_wasRequestedToWaitForAutoInspection = true;
+
+    // This debuggable lives in the UI process, the same process receiving responses from the remote inspector,
```

Comment:
This is only true sometimes. Only Safari has a UI Process that will be asked about inspection, but Service Workers can be originated from other processes as well (a Mac web app, for example)